spec/classes/init_spec.rb

```diff
@@ -12,6 +12,128 @@
+            'poudriere_data'             => '/data',
+            'use_portlint'               => 'yes',
+            'mfssize'                    => '4G',
+            'tmpfs'                      => 'all',
+            'tmpfs_limit'                => 8,
+            'max_memory'                 => 8,
+            'max_files'                  => 1024,
+            'distfiles_cache'            => '/var/cache/distfiles',
+            'svn_host'                   => 'svnmirror.lan',
+            'check_changed_options'      => 'verbose',
+            'check_changed_deps'         => 'no',
+            'bad_pkgname_deps_are_fatal' => 'yes',
+            'pkg_repo_signing_key'       => '/etc/ssl/keys/repo.key',
+            'ccache_enable'              => true,
+            'ccache_dir'                 => '/var/cache/ccache',
+            'ccache_static_prefix'       => '/usr/local',
+            'restrict_networking'        => 'no',
+            'allow_networking_packages'  => 'npm-foo npm-bar',
+            'parallel_jobs'              => 42,
+            'prepare_parallel_jobs'      => 7,
+            'save_wrkdir'                => 'yes',
+            'wrkdir_archive_format'      => 'txz',
+            'nolinux'                    => 'yes',
+            'no_force_package'           => 'yes',
+            'no_package_building'        => 'yes',
+            'http_proxy'                 => 'http://10.0.0.1',
+            'ftp_proxy'                  => 'ftp://10.0.0.2',
+            'no_restricted'              => 'yes',
+            'allow_make_jobs'            => 'yes',
+            'allow_make_jobs_packages'   => 'pkg ccache py*',
+            'timestamp_logs'             => 'no',
+            'url_base'                   => 'http://example.com/poudriere/',
+            'max_execution_time'         => 86400,
+            'nohang_time'                => 7200,
+            'atomic_package_repository'  => 'yes',
+            'commit_packages_on_failure' => 'yes',
+            'keep_old_packages'          => 'yes',
+            'keep_old_packages_count'    => 5,
+            'porttesting_fatal'          => 'yes',
+            'builder_hostname'           => 'pkg.FreeBSD.org',
+            'preserve_timestamp'         => 'yes',
+            'build_as_non_root'          => 'yes',
+            'portbuild_user'             => 'nobody',
+            'portbuild_uid'              => 65534,
+            'priority_boost'             => 'pypy openoffice',
+            'buildname_format'           => '%FT%TZ',
+            'duration_format'            => '%H:%M:%S',
+            'use_colors'                 => 'yes',
+            'trim_orphaned_build_deps'   => 'yes',
+            'local_mtree_excludes'       => '/usr/obj /var/tmp/ccache',
+            'html_type'                  => 'hosted',
+            'html_track_remaining'       => 'yes',
+          }
+        end
+
+        it { is_expected.to compile }
```

Comment:
We can also ensure that not only this class, but the whole cataloge compiles:
```suggestion
        it { is_expected.to compile.with_all_deps }
```